cht/adapter.py

```diff
@@ -108,16 +102,9 @@ def __init__(self, precice, participantName, names, MESH, SHMESH, MODEL, MAT, is
 		self.preciceNodeIndices = []
 		self.preciceFaceCenterIndices = []
 
-		self.preciceFaceCentersMeshID = 0
-		self.preciceNodesMeshID = 0
+		self.readData = dict()
 
-		self.readHCoeffDataID = 0
-		self.readTempDataID = 0
-		self.writeHCoeffDataID = 0
-		self.writeTempDataID = 0
-
-		self.writeTemp = []
-		self.writeHCoeff = []
+		self.writeDataNames = []
```

Comment:
The adapter either writes temperature or the heat coefficient. This can be merged into one one variable. I see no reason why we need to have two lists for the data.